### PR TITLE
docs: reapply "docs: installation tabs code tabs (#1941)"

### PR DESCRIPTION
**Pull request by bennypowers**
This reverts commit 5123961c301525efb20d56b89edeb8e23d452c2b.

### CODE REVIEW

File: docs/_data/repoStatus.yaml [hunk 1]
```diff
@@ -231,7 +231,7 @@
       status: N/A
     - name: Documentation
       status: Ready
-- tagName: rh-navigation-primary      
+- tagName: rh-navigation
```
Comment (zeroedin) [inline docs/_data/repoStatus.yaml:234]: I thought we had discussed at one point that the `rh-navigation` would still have the `-primary` on it?  maybe I'm mistaken. @markcaron

File: docs/_includes/partials/component/footer.njk [hunk 2]
```diff
@@ -1,3 +1,8 @@
+<script data-helmet type="module">
+  import '@rhds/elements/rh-footer/rh-footer-universal.js';
+  import '@rhds/elements/rh-footer/rh-footer-copyright.js';
+</script>
```
Comment (zeroedin) [inline docs/_includes/partials/component/footer.njk:4]: These kinds of changes that move these deps to the file they are needed in are 👨‍🍳 💋

**Review by zeroedin — APPROVED** [review]
Little Glitches Trigger Mayhem

File: docs/_data/repoStatus.yaml (hunk 1 above)
Comment (markcaron) [reply docs/_data/repoStatus.yaml:234]: @zeroedin I believe that was the decision, yeah.

**Pull request merged**